app/code/Magento/InventoryCatalog/Test/Api/StockSourceLink/AssignSourcesToStockTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Test\Api\StockSourceLink;
+
+use Magento\Framework\Webapi\Exception;
+use Magento\Framework\Webapi\Rest\Request;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+class AssignSourcesToStockTest extends WebapiAbstract
+{
+    /**#@+
+     * Service constants
+     */
+    const RESOURCE_PATH_ASSIGN_SOURCES_TO_STOCK = '/V1/inventory/stock/assign-sources';
+    const SERVICE_NAME_ASSIGN_SOURCES_TO_STOCK = 'inventoryApiAssignSourcesToStockV1';
+    /**#@-*/
+
+    /**
+     * @magentoApiDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/sources.php
+     * @magentoApiDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/stock.php
+     * @param string|array $sourceIds
```

Comment:
Looks like in our case `$sourceIds` can be the only array and `$stockId` can be the only int
Pls fix docblock and add type hinting for parameters
Thanks